### PR TITLE
Fix docs rendering

### PR DESCRIPTION
**Pull request by wendigo**
Fixes following rendering problems:

Screenshot 2021-03-03 at 10 55 16
Screenshot 2021-03-03 at 10 53 49
Screenshot 2021-03-03 at 10 54 28
Screenshot 2021-03-03 at 10 54 49

Idk why ``ROLE``s does not render properly so I reworded that section to match the structure in the other `show-.rst` files.

### CODE REVIEW

File: docs/src/main/sphinx/connector/kudu.rst [hunk 1]
```diff
@@ -37,8 +37,8 @@ replacing the properties as appropriate:
    ## For more details see connector documentation.
    #kudu.schema-emulation.enabled=false
 
-   ## Prefix to use for schema emulation (only relevant if `kudu.schema-emulation.enabled=true`)
-   ## The standard prefix is `presto::`. Empty prefix is also supported.
+   ## Prefix to use for schema emulation (only relevant if kudu.schema-emulation.enabled=true)
```
Comment (findepi) [inline docs/src/main/sphinx/connector/kudu.rst:40]: This should use ``` `` ```

File: docs/src/main/sphinx/sql/show-role-grants.rst [hunk 2]
```diff
@@ -12,5 +12,5 @@ Synopsis
 Description
 -----------
 
-List non-recursively the ``ROLE``s that have been granted to the session user
-in ``catalog``, or the current catalog if ``catalog`` is not specified.
+List non-recursively the roles that have been granted to the current user in the specified ``catalog``,
```
Comment (findepi) [inline docs/src/main/sphinx/sql/show-role-grants.rst:15]: "the specified catalog" without ``` `` ```?

cc @mosabua

**Review by losipiuk — APPROVED** [review]

File: docs/src/main/sphinx/connector/kudu.rst (hunk 1 above)
Comment (losipiuk) [reply docs/src/main/sphinx/connector/kudu.rst:40]: If so then also above i assume

Comment (wendigo) [reply docs/src/main/sphinx/connector/kudu.rst:40]: This is a code sample so it's not rendered either way and looks kind of weird.

File: docs/src/main/sphinx/sql/show-role-grants.rst (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/sql/show-role-grants.rst:15]: sure .. its not a code snippet..

**Review by mosabua — APPROVED** [review]
Provided this renders fine when testing locally this looks good. Let me know if you need me to confirm by testing myself..

**Comment by wendigo** [comment]
@mosabua it does render correctly

**Comment by mosabua** [comment]
Great.. merge away ;-)

**Comment by findepi** [comment]
Any reason for SHOW ROLES AND SHOW ROLE GRANTS to use different wording?

image

**Pull request closed without merging**